docs/aks-acr-across-tenants.md

```diff
@@ -0,0 +1,59 @@
+# AKS and ACR in different tenants
+
```

Comment:
Maybe a title like "Set up AKS to pull from ACR in a different AD tenant"?

Reply:
updated